mantidproject/mantid
# Add period selection to the ISIS SANS GUI

~~**Do not merge or test before #20229 has been merged**~~

This PR adds a period selection to the data table. 
Fixes #20287

**To test:**

Please find the test data here: smb://olympic/babylon5/Scratch/Anton/LARMOR_MULTI_PERIOD_REDUCTION

1. Open the ISIS SANS v3 experimental interface
2. Set the instrument to LARMOR
3. Load the user file: USER_Bender_RB1610214_162DMagnet_parallel_r10182.txt
4. In the data table set `LARMOR00013509` for the Sample scatter entry
5. Press `Process`
   * Confirm that a WorkspaceGroup with **four** reduced workspaces has been produced. 
6. Delete the output workspaces and the loaded workspaces
6. In the column `ssp` set 2 next to the scatter data entry
5. Press `Process`
   * Confirm that a single reduced workspace was generated (it should have `p2` in the name)


**Release Notes** 
Does not have to be in release notes, since the GUI is released in this cycle.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
